packages/zowe-explorer-api/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@zowe/zowe-explorer-api",
-  "version": "1.17.0",
+  "version": "1.17.0-next.1",
```

Comment:
In case we don't want to be changing this value manually, we could run `npm version prerelease --preid=next` for this package 😋 
Or document it in the wiki after this PR is merged.

Reply:
@zFernand0 all PRs into the next branch will have a version change, similar to the next releases of Zowe CLI that includes the date which would kick off a deploy.  For example:
the next branch: https://github.com/zowe/vscode-extension-for-zowe/blob/ae1c806eb434bb7c17829e78e15d13c1adfcda69/package.json#L3
[next] Support project level config PR: https://github.com/zowe/vscode-extension-for-zowe/blob/d6fcfd59c6cdc1183eeb9f1f0291ab08c66ecc96/package.json#L3

What we could do, similar to our master/release process is not to update the version name until ready to publish the next version